src/components/productDetails/list.jsx

```diff
@@ -1,41 +1,70 @@
-import React, { useEffect } from "react";
+import React, { useState, useEffect } from "react";
+import {
+  fetchOutfits,
+  fetchOutfitsImages,
+  addToOutfit,
+  removeFromOutfit
+} from "./outfitHelpers.js";
 import Card from "./card.jsx";
+import AddToOutfitCard from "./addCard.jsx";
 
 const List = ({
   listName,
   pageProduct,
   products,
   productsImages,
   onClickDetails,
-  fetchRelatedDataAsync,
+  fetchRelatedDataAsync
   //avgRating,
-  addToOutfit, // FROM MAP DISPATCH TO STORE
-  removeFromOutfit // FROM MAP DISPATCH TO STORE
 }) => {
+  var [outfits, setOutfits] = useState([]);
+  var [outfitsImages, setOutfitsImages] = useState([]);
+
+  if (listName === "Outfit") {
+    products = outfits;
+    productsImages = outfitsImages;
+  }
   //HANDLE FETCHING DATA ON AFTER FIRST RENDER
   useEffect(() => {
-    fetchRelatedDataAsync(1);
+    if (listName === "Related") {
+      fetchRelatedDataAsync(4);
```

Comment:
We'll all have to figure out how to move away from hard-coding product IDs as we have a better idea about how the store will be initialized.

Reply:
Yes i agree. Something we will need to keep in mind.